src/packages/migrate/src/__tests__/DbSeed.test.ts

```diff
@@ -106,8 +106,8 @@ Please provide the --preview-feature flag to use this command.
     const result = DbSeed.new().parse(['--preview-feature'])
     await expect(result).resolves.toMatchInlineSnapshot(`
 
-                                                🌱  The seed command has been executed.
-                                        `)
+                                                                        🌱  The seed command has been executed.
+                                                            `)
```

Comment:
Something is wrong about the snapshot indentation again. This will keep happening it seems :/

Reply:
Yes, it's a recurrent thing we don't know why, "Jest"...

Reply:
You can just ignore these change and revert the lines so that the snapshots keeps making sense. They still match just fine.